reference/recency-surveillance.v1.json

```diff
@@ -360,7 +365,7 @@
                         ]
                       },
                       "presumtive_tb": {
```

Comment:
@rkiranda Can you change this typo as well? You changed the description but not the Key